platforms/ios/example/WysiwygUITests/WysiwygUITests+InlinePredictiveText.swift

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright 2024 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import XCTest
+
+extension WysiwygUITests {
+    func testInlinePredictiveText() {
+        sleep(5)
+        app.typeTextCharByCharUsingKeyboard("Hello how")
+        app.keys["space"].tap()
+        sleep(1)
+        app.keys["space"].tap()
+        assertTextViewContent("Hello how are you ")
+    }
+    
+    func testInlinePredictiveTextIgnored() {
+        sleep(5)
+        app.typeTextCharByCharUsingKeyboard("Hello how")
+        button(.sendButton).tap()
+        sleep(1)
+        assertContentText(plainText: "Hello how", htmlText: "Hello how")
+    }
+    
+    func testDoubleSpaceIntoDot() {
+        sleep(5)
+        app.typeTextCharByCharUsingKeyboard("Hello")
+        app.keys["space"].tap()
+        app.keys["space"].tap()
+        assertTextViewContent("Hello. ")
+    }
+}
```

Comment:
@langleyd added these UI tests to check if our most important cases are covered and they work very well.... however the predictive text test may be a bit flaky if the simulator is not recently initialised, which luckily should not be an issue on the CI

Reply:
Are there any other test cases we may want to cover that this PR addresses? 
(I am also looking into finding a way to test CJK, but seems there is no reliable way to simulate a keyboard suggestion tap, will look more into it)

Reply:
No these cover the ones I can think of without the software keyboard.